models/models.py

```diff
@@ -0,0 +1,75 @@
+
+
+
+
+
+
+#User Model in SQL
+class User(db.Model):
+    
+    __table__ = "Users"
+
+    id = db.Column(db.Integer, primary_key=True)    
+    username = db.Column(db.String(50))
+    email = db.Column(db.String(60))
+    password = db.Column(db.String(80))
+
+
+    def __init__(self,username,email, password):
+        #initiliazing User class constructor
+        self.username=username
+        self.email = email
+        self.password = password
+
+
+
+    def create(self):
+        #method to create a new record in the databse
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        #method to make changes to a record in the datase
+        db.session.commit()
+
+    def delete(self):
+        #method to delete record in database
+        db.session.delete()
+        db.session.commit()
+    
+
+
+
+    
+
+class Recipe(db.Model):
+    
+    __table__ = "Recipes"
+
+    recipe_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(30))
+    description = db.Column(db.String(1000))
+    id = db.Column(db.Integer)
+
```

Comment:
#### how to do relationships
if User can have many Recipes, define a *relationship field* in User and a *ForeignKey* field in Recipe.
e.g.
```python
class User:
    ....
    recipes = db.relationship(...)

class Recipe:
    ....
    user_id = db.Column(db.Integer, db.ForeignKey('user.id')
```